### PR TITLE
fix:[CDS-44453]:canvas-nodes-cypress-assertion

### PR DESCRIPTION
**Pull request by namanharness**
### Summary

Asserting Stages on the Canvas when its resized with 
- `Resizer horizontal `
- Zoom In/Out
- Dragging the last stage of pipeline.
#### Screenshots

https://user-images.githubusercontent.com/110456765/194883412-8c46814c-2395-499e-9927-7777ea4dd7bb.mov

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: cypress/integration/70-pipeline/CanvasStagesAssertion.spec.ts [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  featureFlagsCall,
+  canvasExecutionPipelineCall,
+  canvasExecutionStageNodeCall
+} from '../../support/70-pipeline/constants'
+
+describe('Canvas Stages Assertion', () => {
+  //function to drag element vertically
+  function dragElementVerticallyBy(el, dragAmount) {
+    const rect = el[0].getBoundingClientRect()
+    const pageYDragAmount = dragAmount
```
Comment (PratyushGar) [inline cypress/integration/70-pipeline/CanvasStagesAssertion.spec.ts:20]: Variable assignment is not needed, directly use `dragAmount`

File: cypress/integration/70-pipeline/CanvasStagesAssertion.spec.ts [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+            name: 'NG_SVC_ENV_REDESIGN',
+            enabled: true,
+            lastUpdatedAt: 0
+          }
+        ]
+      }).as('enableFeatureFlag')
+      cy.initializeRoute()
+    })
+  })
+  it.only('Parallel Nodes Pipeline Assertion', () => {
```
Comment (PratyushGar) [inline cypress/integration/70-pipeline/CanvasStagesAssertion.spec.ts:67]: `.only` is not required

File: cypress/integration/70-pipeline/CanvasStagesAssertion.spec.ts (hunk 1 above)
Comment (namanharness) [reply cypress/integration/70-pipeline/CanvasStagesAssertion.spec.ts]: Resolved 👍

File: cypress/integration/70-pipeline/CanvasStagesAssertion.spec.ts (hunk 2 above)
Comment (namanharness) [reply cypress/integration/70-pipeline/CanvasStagesAssertion.spec.ts]: Resolved 👍

**Review by PratyushGar — APPROVED** [review]

**Review by vikrant-harness — APPROVED** [review]

**Pull request merged**